test/platforms/ios/index.html

```diff
@@ -0,0 +1,52 @@
+<style>
+body {
+  /* Use an off white color to provide contrast around saturated output
+   pixels */
+  background-color: black;
+  color: white;
+  font-family: arial;
+}
+
+/* Highlight error messages */
+.error {
+  color: red;
+}
+
+/* Draw data, like printed images with a small font. */
+.data {
+  font-size: 4pt;
+}
+
+/* Outline displayed images so that their extent is clearly defined. */
+.output {
+  border-color: rgb(240,240,250);
+  border-style: solid;
+  border-width: 5px;
+}
+
+</style>
+
+Very simple Halide app<br>
+
+<script type="text/javascript">
+
+// These variables are used to configure app behavior. They are read by the app
+// after the page loads.
+
+// String containing a list of comma deliminated symbol names to be loaded and
+// executed by the app.
+var AppTestSymbols = "test_example_generator";
```

Comment:
Good catch. That functionality was replaced by the test_symbols.h.template below. Previously the app would look this symbol up in the java script context and get the list of symbols to test. That way you could swap html files and test different behavior. PNACL didn't support dlsym, so that mechanism had to be removed.